File: WarehouseManagementWeb.Infrastructure/Identity/Models/UserSignUpInput.cs
namespace WarehouseManagementWeb.Infrastructure.Identity.Models
{
    /// <summary>
    /// Класс входной модели регистрации пользователя.
    /// </summary>
    public class UserSignUpInput
    {
        /// <summary>
        /// Имя пользователя.
        /// </summary>
        public string? FirstName { get; set; }

        /// <summary>
        /// Фамилия пользователя.
        /// </summary>
        public string? LastName { get; set; }

        /// <summary>
        /// Email пользователя.
        /// </summary>
        public string? Email { get; set; }

        /// <summary>
        /// Пароль пользователя.
        /// </summary>
        public string? Password { get; set; }

        /// <summary>
        /// Подтверждение пароля.
        /// </summary>
        public string? PasswordConfirm { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/auth/models/input/user-sign-up-input.ts
/**
 * Класс входной модели регистрации пользователя.
 */
export class UserSignUpInput {
  /**
   * Имя пользователя.
   */
  firstName: string = '';

  /**
   * Фамилия пользователя.
   */
  lastName: string = '';

  /**
   * Email пользователя.
   */
  email: string = '';

  /**
   * Пароль пользователя.
   */
  password: string = '';

  /**
   * Подтверждение пароля.
   */
  passwordConfirm: string = '';
}
